modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -21,6 +21,10 @@ definition {
 			Click(
 				key_itemName = "Feature Flags",
 				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			Click(
+				key_menuBar = "Release",
+				locator1 = "FeatureFlag#MENU_BAR");
```

Comment:
Please replace lines 16 thru 27 with PortalSettings.gotoConfiguration(). We don't need to add more paths.